final_task/rss_reader/Entry.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+
+class Entry:
+    """class for every article from http:link...link.rss"""
+
+    def __init__(self, title: str, date: str, article_link: str, summary: str, links: list):
+        self.title = title
+        self.date = date
+        self.article_link = article_link
+        self.summary = summary
+        self.links = links
+        self.parse_html()
+        logging.info("Entry object created")
+
+    def print_title(self) -> None:
+        logging.info("function \"print_title\" started")
+
+        print("Title: ", self.title)
+
+        logging.info("function \"print_title\" finished")
+
+    def print_date(self) -> None:
+        logging.info("function \"print_date\" started")
+
+        print("Date: ", self.date)
+
+        logging.info("function \"print_date\" finished")
```

Comment:
Here, above and below. You also can make these lines better (logging part). Think about actions you perform here:

1. You perform some logic (write logs) before performing function itself.
2. You perform function.
3. You perform some logic (write logs) after function.

We have a potential for a good decorator here. It is perfect example of decorator usage.

Reply:
fixed